pkg/tss/capabilities.go

```diff
@@ -73,3 +74,64 @@ func readTPM20VendorAttributes(rwc io.ReadWriter) (TPMInfo, error) {
 		FirmwareVersionMinor: int(fw.Value & 0x0000ffff),
 	}, nil
 }
+
+func takeOwnership12(rwc io.ReadWriteCloser, ownerPW, srkPW string) (bool, error) {
+	var ownerAuth [20]byte
```

Comment:
Is this 20 bytes zero?

Reply:
What do you mean? Does it matter here? I use it to store a hash value of sha1 (160bit = 20 byte).
Please elaborate

Reply:
you need to define the WELL_KNOWN_SECRET which is 20 bytes of zero.

Reply:
[20]byte initializes with default of zeros. Should be fine.